Windows CI seems broken

When I submit a PR (#72), the windows ci failed:
```
Starting: Bash
==============================================================================
Task         : Bash
Description  : Run a Bash script on macOS, Linux, or Windows
Version      : 3.171.1
Author       : Microsoft Corporation
Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/bash
==============================================================================
Generating script.
C:\Windows\system32\bash.exe --noprofile --norc -c pwd
Windows Subsystem for Linux has no installed distributions.


Distributions can be installed by visiting the Microsoft Store:


https://aka.ms/wslstore



##[error]The process 'C:\Windows\system32\bash.exe' failed with exit code 4294967295
```

I think this has nothing to do with my PR, but an outdated ci config.

According to @Et7f3 on [Discord](https://discordapp.com/channels/235176658175262720/235200837608144898/742159260485419016) this can be fixed by cherry-picking a [commit from the reason repo](https://github.com/facebook/reason/commit/d3822b78b9a8c37594d3d5bff640f9159af0e298).

--- a/.ci/build-platform.yml
+++ b/.ci/build-platform.yml
@@ -16,10 +16,10 @@ jobs:
       # Needed so that the mingw tar doesn't shadow the system tar. See
       # pipelines.yaml. We need windows bsdtar from system32, not the mingw
       # one. Note powershell doesn't need escaping of backslashes.
-      - powershell: Write-Host "##vso[task.setvariable variable=PATH;]C:\Windows\system32;${env:PATH}"
+      - powershell: Write-Host "##vso[task.setvariable variable=PATH;]C:\Program Files\Git\bin;C:\Windows\system32;${env:PATH}"
         continueOnError: true
         condition: eq(variables['AGENT.OS'], 'Windows_NT')
-        displayName: "Make sure windows/system32 is at front of path if windows"
+        displayName: "Make sure C:/Program Files/Git/bin and windows/system32 is at front of path if windows"
       - powershell: $Env:Path
         continueOnError: true
         condition: and(eq(variables['AGENT.OS'], 'Windows_NT'), and(eq(variables['Build.Reason'], 'PullRequest'), and(succeeded(), ne(variables['Build.SourceBranch'], variables['System.PullRequest.TargetBranch']))))

